tests/kernel_bundle/sycl_link_different_device_for_sycl_link.cpp

```diff
@@ -0,0 +1,70 @@
+/*******************************************************************************
+//
+//  SYCL 2020 Conformance Test Suite
+//
+//  This test calls sycl::link(vector<kernel_bundle<>>, std::vector<device>,
+//  property_list) and sycl::link(kernel_bundle<>, std::vector<device>,
+//  property_list). For obtained kernel bundle with object state will use
+//  one device, but for sycl::link will be provided vector with device, that was
+//  used for kernel bundle and different devices.
+//
+//  The test verifies that an exception with sycl::errc::invalid was thrown.
+//
+*******************************************************************************/
+
+#include "sycl_link.h"
+#include "../common/assertions.h"
+
+#define TEST_NAME sycl_link_different_device_for_sycl_link
+
+namespace TEST_NAMESPACE {
+using namespace sycl_cts;
+using namespace kernels;
+using namespace sycl_cts::tests::sycl_link;
+using namespace sycl_cts::tests::kernel_bundle;
+
+class TEST_NAME : public sycl_cts::util::test_base {
+ public:
+  /** return information about this test
+   */
+  void get_info(test_base::info &out) const override {
+    set_test_info(out, TOSTRING(TEST_NAME), TEST_FILE);
+  }
+
+  /** execute the test
+   */
+  void run(util::logger &log) override {
+    const std::vector<sycl::device> devices{sycl::device::get_devices()};
+    if (devices.size() <= 1) {
+      log.note("Test skipped due to one device was found");
```

Comment:
Cf previous PR on this very same note message

Reply:
I've applied this comment in this commit: https://github.com/KhronosGroup/SYCL-CTS/pull/227/commits/050ffa8abb11c4f5b529f960904b20337a8b10e1